src/Randomizer.App/Windows/RomListWindow.xaml.cs

```diff
@@ -594,5 +605,39 @@ private void CopyTextToClipboard(string text)
                 Clipboard.SetDataObject(text);
             }
         }
+
+        private void ProgressionLogMenuItem_Click(object sender, RoutedEventArgs e)
+        {
+            if (sender is not MenuItem menuItem)
+                return;
+
+            if (menuItem.Tag is not GeneratedRom rom)
+                return;
+
+            if (rom.TrackerState == null)
+            {
+                MessageBox.Show(this, "There is no history for the selected rom.",
+                        "SMZ3 Cas’ Randomizer", MessageBoxButton.OK, MessageBoxImage.Warning);
+                return;
+            }
+
+            _dbContext.Entry(rom.TrackerState).Collection(x => x.History).Load();
+
+            if (rom.TrackerState.History == null || rom.TrackerState.History.Count == 0)
```

Comment:
Would it be possible to disable the option if you right-click on a rom without history?